### PR TITLE
Adds kaboom debug command for iterative explosions

### PR DESCRIPTION
**Pull request by quardbreak**
## Description of changes

Adds kaboom debug command for iterative explosions

## Why and what will this PR improve

kaboom

## Authorship

Lohikar

### CODE REVIEW

File: code/modules/admin/verbs/debug.dm [hunk 1]
```diff
@@ -516,3 +516,10 @@
 
 	new_planet.update_daynight()
 	new_planet.build_level()
+
+/client/proc/kaboom()
+	set name = "kaboom"
+	set category = "Debug"
+	var/power = input(src, "Input power num.", "Power?") as num
+	var/turf/T = get_turf(mob)
+	explosion_iter(T, power)
```
Comment (Lohikar) [inline code/modules/admin/verbs/debug.dm]:
```suggestion
	explosion_iter(T, power, UP|DOWN)
```

**Comment by MistakeNot4892** [comment]
Add an iterative yes/no check to Drop-Bomb rather than a new, much less feature rich verb please.

**Review by MistakeNot4892 — APPROVED** [review]
Nobody seems to object to this but me so w/e.

**Comment by afterthought2** [comment]
I mean, I agree with you: one verb suffices for this explosion stuff.

**Comment by MistakeNot4892** [comment]
Given this is conflicted, stale, and somewhat redundant in view of the other explosion PR, I'm gunna close this.

**Pull request closed without merging**